examples/jsm/effects/AsciiEffect.js

```diff
@@ -118,8 +118,8 @@ class AsciiEffect {
 		}
 
 
-		const aDefaultCharList = ( ' .,:;i1tfLCG08@' ).split( '' );
-		const aDefaultColorCharList = ( ' CGO08@' ).split( '' );
```

Comment:
I wonder why the split was in use in the first place 🤔 .

Reply:
Probably because the implementation expects an array with chars but using `.length` and `[]` access also works for strings. In this case, it's maybe better to always use strings.

Reply:
On the other side, working with array might be faster. How about we change the line:
```js
if ( charSet ) aCharList = charSet;
```
to
```
if ( charSet ) aCharList = charSet.split( '' );
```
In this way, the code still uses arrays and we don't risk any kind of (performance) regressions.